app/Extensions/Avatar/Schemas/AppLogoSchema.php

```
@@ -0,0 +1,34 @@
+{
+    public function getId(): string
+    {
+        return 'appIcon';
+    }
+
+    public function getName(): string
+    {
+        return 'App Icon';
+    }
+
+    public function get(User $user): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)